feature-libs/product/variants/assets/translations/en/index.ts

```diff
@@ -1,5 +1,5 @@
 import { productVariants } from './variants.i18n';
 
 export const en = {
-  productVariants,
+  variants: productVariants,
```

Comment:
Add information about removed `variant` keys in 4_0.md doc and it's new replacement `variants` in separate library and entrypoint.